api/spec/controllers/mno_enterprise/jpi/v1/admin/app_instances_controller_spec.rb

```diff
@@ -4,30 +4,35 @@
 module MnoEnterprise
   describe Jpi::V1::Admin::AppInstancesController, type: :controller do
     include MnoEnterprise::TestingSupport::SharedExamples::JpiV1Admin
-    #TODO: Fix Spec for Admin Controller
-    before { skip }
 
     render_views
     routes { MnoEnterprise::Engine.routes }
-    before { request.env["HTTP_ACCEPT"] = 'application/json' }
+    before { request.env['HTTP_ACCEPT'] = 'application/json' }
+    before { stub_audit_events }
 
     let(:user) { build(:user, :admin, :with_organizations) }
+    let!(:current_user_stub) { stub_user(user) }
```

Comment:
Why the`let!` and not just:
```
before do
  stub_user(user)
  sign_in user
end
```

Reply:
Same for the other spec files

Reply:
For multiple reasons

- to be sure it was unique
- eventually to be tested (it has been called)
- to be more semantic